bot/cogs/fun.py

```diff
@@ -92,10 +160,13 @@ async def image(self, ctx: Context, type: str) -> None:
             )
             embed.set_image(url=nekos.img(type))
             await ctx.send(embed=embed)
+
         except errors.NSFWChannelRequired:
             await ctx.send("Hey dude! Go use this command in a NSFW Channel, this ain't ur home.")
+
         except nekos.errors.InvalidArgument:
             await ctx.send(f"Invalid type! Possible types are : ```{', '.join(config.nsfw_possible)}```")
+
```

Comment:
Usually, excepts aren't separated by newlines

Reply:
I agree, this is janas code, I'll update it.